### ISSUE TITLE
PlatformException(error, You must call this method on a background thread, null)

### ISSUE THREAD

**Issue by msarkrish** [bumptech/glide#4328]
**Glide Version**:
4.11.0

**Integration libraries**:
No

**Device/Android Version**:
Redmi Note 4/Nougat

**Issue details / Repro steps / Use case background**: 
PlatformException(error, You must call this method on a background thread, null)

**Glide load line / `GlideModule` (if any) / list Adapter code (if any)**:
```java
val bitmap: Bitmap =Glide.with(context)
                 .asBitmap()
                 .load(filePath)
                 .submit()
                 .get()
```

**Layout XML**:
I'm using for Flutter Application, via Platform method. not for Native Android

I'm followed your documentation, for background thread,
https://bumptech.github.io/glide/doc/getting-started.html#background-threads

In my app i'm showing all the videos from device storage. 

**Stack trace / LogCat**:
```ruby

```

**Comment by msarkrish** [comment]
Anyone know how to solve this Issue?

**Comment by lifeover23** [comment]
What's with my husband Quit the barn with it. It already has a wife. Send a lot of mail like this What's wrong with my husband? Will you make an appointment for a video clip with sex? You guys are bad. My husband is damn As for you, it's bad, go die, my son should have a good father. There should be no bad dads like this at all.
Want to talk privately Call me mail Dew4440@gmail.com
I will wait to talk

ส่งจากอุปกรณ์ Samsung ส่วนตัว

-------- ข้อ​ความ​ดั้ง​เดิม --------
จาก: "stale[bot]" <notifications@github.com> 
วัน​ที่: 4/10/2020  10:12  (GMT+07:00) 
ถึง: bumptech/glide <glide@noreply.github.com> 
Cc: Subscribed <subscribed@noreply.github.com> 
เรื่อง: Re: [bumptech/glide] PlatformException(error, You must call this
  method on a background thread, null) (#4328) 


Closed #4328.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/bumptech/glide/issues/4328#event-3837160261",
"url": "https://github.com/bumptech/glide/issues/4328#event-3837160261",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by mejdiHF** [comment]
@lifeover23 what
